packages/ui/src/components/Icons/PrivacyIcon.tsx

```diff
@@ -1,16 +1,24 @@
+import classNames from 'classnames';
 import React, { SVGProps } from 'react';
 
 import Icon from '@/assets/icons/privacy-icon.svg';
 
+import styles from './PrivacyIcon.module.scss';
+
 type Props = {
   type?: 'show' | 'hide';
 } & SVGProps<SVGSVGElement>;
 
 const PrivacyIcon = ({ type = 'show', ...rest }: Props) => {
   return (
-    <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" {...rest}>
-      <use href={`${Icon}#${type}`} />
-    </svg>
+    <span className={classNames(styles[type], styles.privacyIcon)}>
+      <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" {...rest}>
+        <use href={`${Icon}#show`} />
+      </svg>
```

Comment:
i think it's the `use` issue to load svg every time. how about using a react component

Reply:
hmm let me check